spec.bs

```diff
@@ -134,33 +192,99 @@ Modify {{Document/requestStorageAccess()}} at step 14.1.1.1.1 to read:
 
 <h3 id="storage">Changes to various client-side storage mechanisms</h3>
 
-<h4 id="dom-storage">DOM Storage</h4>
+For all of the following getters and methods, consider the |docForSAA| to be |doc| with the following modifications:
```

Comment:
```suggestion
For all of the following getters and methods, consider the following modifications:
```

Reply:
I'm not sure how the document plays into this, so I would suggest removing it for now but filing a new issue for wiring up a "unpartitioned" bit into the `[=obtain a storage key=]` algorithm, I think that's actually not that hard, we just have to trace it from all the getters monkeypatched below.

In the meantime, I think this is clear enough.